qa/tasks/scripts/ses_disk_fault_injection.sh

```diff
@@ -9,8 +9,12 @@ function health_ok() {
     done
 }
 
-storage_device_name=$(salt $disk_storage_minion cmd.run \
- "pvdisplay | grep -B 1 'VG Name .* ceph' | egrep -v 'ceph-block-dbs|nvme|--' | head -1 | cut -d / -f 3" --output=json | jq -r .[])
+minion=${disk_storage_minion%%.*} # target-ses-097.ecp.suse.de -> target-ses-097
+random_osd=$(ceph osd tree | grep -A 1 $minion | grep -o "osd\.".* | awk '{print$1}')
```

Comment:
What makes the the osd random? Can you give an example how the `ceph osd tree` look like so we are sure this line is doing right thing.

Reply:
It's like:
```
 ceph osd tree
ID CLASS WEIGHT  TYPE NAME               STATUS REWEIGHT PRI-AFF
-1       0.02939 root default
-3       0.00980     host target-ses-039
 0   hdd 0.00980         osd.0               up  1.00000 1.00000
-5       0.00980     host target-ses-040
 1   hdd 0.00980         osd.1               up  1.00000 1.00000
-7       0.00980     host target-ses-060
 2   hdd 0.00980         osd.2             down  1.00000 1.00000
```

and script itself is run like: 
```
INFO:tasks.ses_qa.validation:(scripts) running script ses_disk_fault_injection.sh with args ['target-ses-060.ecp.suse.de', 'target-ses-040.ecp.suse.de'] on remote target-ses-058.ecp.suse.de
``` 
and based on `declare -a disk_storage_minion=$1` we use the first entry of the array.

Reply:
Thanks, you can add this as an example to the source code in comments, so it is clear what do you parse.
Alos, instead of `grep -o "osd\.".* | awk '{print$1}'` you could use `grep -o "osd\.[^\s+]"`